profiles/FlimsyLC - RUN v1.0.json

```diff
@@ -0,0 +1,83 @@
+{
+  "id": 17,
+  "name": "Flimsy LC RUN",
```

Comment:
Pls pay attention to the [profile creation guidelines](https://github.com/Zer0-bit/gaggiuino/tree/community/profiles#creating-a-profile)